schemas/2015-01-01/deploymentTemplate.json

```diff
@@ -2077,13 +2197,7 @@
                   { "$ref": "https://schema.management.azure.com/schemas/2019-06-01-preview/Microsoft.Synapse.json#/resourceDefinitions/workspaces_sqlPools_transparentDataEncryption" },
                   { "$ref": "https://schema.management.azure.com/schemas/2019-06-01-preview/Microsoft.Synapse.json#/resourceDefinitions/workspaces_sqlPools_vulnerabilityAssessments" },
                   { "$ref": "https://schema.management.azure.com/schemas/2019-06-01-preview/Microsoft.Synapse.json#/resourceDefinitions/workspaces_sqlPools_vulnerabilityAssessments_rules_baselines" },
-                  { "$ref": "https://schema.management.azure.com/schemas/2018-09-01-preview/Microsoft.ResourceGraph.json#/resourceDefinitions/queries" },
-                  { "$ref": "https://schema.management.azure.com/schemas/2016-07-12-preview/Microsoft.Advisor.json#/resourceDefinitions/recommendations_suppressions" },
```

Comment:
Why are these being removed? May be good reason, but I'm not seeing it. I've spot checked a few of these to confirm that the schemas still exist.

Reply:
All of these URLs 404 - I can add them back if they're like supposed to be published or something (but haven't been yet?)

Reply:
Ah, yep, I see it. 

@jorgecotillo I do not know the publishing end of this very well. I can see that all of these have schema files in this repo. Do we need to check on the published status for _schema.management.azure.com_?

Perhaps this is a non-issue, if so LGTM.